ci/infra/aws/security-groups.tf

```diff
@@ -102,96 +113,120 @@ resource "aws_security_group" "allow_control_plane_traffic" {
     "Name", "${var.stack_name}-control-plane",
     "Class", "SecurityGroup"))}"
 
+  # etcd - internal
   ingress {
-    from_port   = 2380
+    from_port   = 2379
     to_port     = 2380
     protocol    = "tcp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
 
+  # VXLAN traffic (cilium) - internal
   ingress {
-    from_port   = 8285
-    to_port     = 8285
+    from_port   = 8471
+    to_port     = 8472
     protocol    = "udp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
 
+  # CRI-O streaming - internal only
   ingress {
-    from_port   = 30000
-    to_port     = 32768
-    protocol    = "udp"
+    from_port   = 10010
+    to_port     = 10010
+    protocol    = "tcp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
-}
-
-resource "aws_security_group" "allow_workers_traffic" {
-  description = "give access to some traffic on the workers"
-  name        = "${var.stack_name}-allow-workers-traffic"
-  vpc_id      = "${aws_vpc.platform.id}"
-
-  tags = "${merge(local.tags, map(
-    "Name", "${var.stack_name}-control-plane",
-    "Class", "SecurityGroup"))}"
 
+  # master -> worker kubelet communication - internal
   ingress {
-    from_port   = 80
-    to_port     = 80
+    from_port   = 10250
+    to_port     = 10250
     protocol    = "tcp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
 
+  # kubeproxy health check - internal only
   ingress {
-    from_port   = 443
-    to_port     = 443
+    from_port   = 10256
+    to_port     = 10256
     protocol    = "tcp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
 
+  # range of ports used by kubernetes when allocating services
+  # of type `NodePort` - internal
   ingress {
-    from_port   = 8080
-    to_port     = 8080
+    from_port   = 30000
+    to_port     = 32767
     protocol    = "tcp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
 
+  # range of ports used by kubernetes when allocating services
+  # of type `NodePort` - internal
   ingress {
-    from_port   = 8081
-    to_port     = 8081
-    protocol    = "tcp"
+    from_port   = 30000
+    to_port     = 32767
+    protocol    = "udp"
     cidr_blocks = ["${var.vpc_cidr_block}"]
   }
+}
+
+resource "aws_security_group" "allow_workers_traffic" {
+  description = "give access to some traffic on the workers"
+  name        = "${var.stack_name}-allow-workers-traffic"
+  vpc_id      = "${aws_vpc.platform.id}"
 
+  tags = "${merge(local.tags, map(
+    "Name", "${var.stack_name}-control-plane",
+    "Class", "SecurityGroup"))}"
+
+  # VXLAN traffic (cilium) - internal
   ingress {
-    from_port   = 2380
-    to_port     = 2380
+    from_port   = 8471
+    to_port     = 8472
+    protocol    = "udp"
+    cidr_blocks = ["${var.vpc_cidr_block}"]
+  }
+
+  # CRI-O streaming - internal only
```

Comment:
Do we really need the CRI-O streaming feature exposed?

Reply:
I don't think so, I'm discussing that over there: https://github.com/SUSE/doc-caasp/pull/398

Reply:
Dropped